.github/scripts/publish_container.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+GIT_VERSION=$(git describe --tags --always --match 'v*')
+XYZ_VERSION=${GIT_VERSION%-*-*}
```

Comment:
what happens when GIT_VERSION is `v1.0.0` vs `v1.0.0-12-gdeff9a1`

Reply:
We will create the tag `v1.0.0` twice, don't know how it's handled

Reply:
I've made a test and it does not seem to tag the container twice, event if we run the command twice:
![image](https://user-images.githubusercontent.com/86932794/212181719-1c6ee35d-792c-45f4-b9bb-c875301d550f.png)

If I find a better way to handle the `GIT_VERSION` I'll change that but I think for now that's enough?